packages/net-modules.spec

```diff
@@ -32,13 +29,15 @@ The first implementation in this repository showcases Apache Mesos using Project
 %build
 ./bootstrap
 
-%configure --with-mesos=/usr/include/mesos/ --with-protobuf=/usr
+CPPFLAGS='-I/root/protobuf/protobuf-2.5.0/src/ -I/root/glog/glog-0.3.3/src/ -I/root/boost/boost-1.53.0/' ./configure --with-mesos=/
 make -j 2
 
 %install
 %make_install
 
+rm -f /usr/local/lib/mesos/*.la
```

Comment:
[Standard RPM packaging rules say to remove all static library files](https://fedoraproject.org/wiki/Packaging:Guidelines). I copied this from [our original mesos spec](https://github.com/mesosphere/net-modules/pull/109/files#diff-b6afc46629fd72a7ea1541f1a3ba0dbcL168)